src/components/ProfitAndLossChart/ProfitAndLossChart.tsx

```diff
@@ -174,6 +176,28 @@ export const ProfitAndLossChart = ({
     currentDate: startOfMonth(Date.now()),
   })
 
+  const anyData = useMemo(() => {
+    return Boolean(
+      data?.find(
+        x =>
+          x.income !== 0 ||
+          x.costOfGoodsSold !== 0 ||
+          x.grossProfit !== 0 ||
+          x.operatingExpenses !== 0 ||
+          x.profitBeforeTaxes !== 0 ||
+          x.taxes !== 0 ||
+          x.totalExpenses !== 0,
+      ),
+    )
+  }, [data])
```

Comment:
Not sure how to capture that there is no data... Above, I'm looking for any non-zero value